docker/entrypoint/docker-entrypoint.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
+
+set -e
+
+APP_DIR="/opt/tarantool"
+DATA_DIR="/var/lib/tarantool/sys_env"
+LOG_DIR="/var/log/tarantool/sys_env"
+RUN_DIR="/var/run/tarantool/sys_env"
+
+APP_CONFIG="${APP_DIR}/${TT_APP_NAME}/config.yaml"
+
+if [ -f "${APP_CONFIG}" ] && [ -z "${TT_CONFIG}" ]; then
+    export TT_CONFIG="${APP_CONFIG}"
+else
+    export TT_CONFIG="${TT_CONFIG:-/default-config.yaml}"
+fi
+
+export TT_SNAPSHOT_DIR="${TT_SNAPSHOT_DIR:-${DATA_DIR}/${TT_APP_NAME}/${TT_INSTANCE_NAME}}"
```

Comment:
I think that we should use `TT_*_DEFAULT` variables here, so they can be redefined in a user-provided configuration file or by `TT_*` environment variables (without the `_DEFAULT` suffix).

Reply:
Fixed.